config/olm/openshift/0.2.1/manifests/dynatrace.com_dynakubes.yaml

```diff
@@ -1,21 +1,11 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Please also revert this file.
For 0.2.1 it should stay as it was but for the next release it will get those updates